examples/plotting/plotting_blank_map.py

```diff
@@ -0,0 +1,59 @@
+"""
+=============================
+Plot positions on a blank map
+=============================
+
+This example showcases how to plot positions on a blank map.
+It is commonly seen in papers and presentations to show HPC positions of events that occurred at
+different times and therefore no single observation is appropriate.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.coordinates import frames
+
+################################################################################
+# First we will create a blank map using with an array of zeros.
+# Since there is no WCS information, we will need to construct a header to pass to Map.
+data = np.full((1000, 1000), 0)
+
+# Define coordinates and frame of reference and make the header using sunpy.map.make_fitswcs_header
+skycoord = SkyCoord(0*u.arcsec, 0*u.arcsec, obstime='2013-10-28',
+                    observer='earth', frame=frames.Helioprojective)
+# Scale set to the following for solar limb to be in the field of view
+header = sunpy.map.make_fitswcs_header(data, skycoord, scale=[2, 2]*u.arcsec/u.pixel)
+
+# Use sunpy.map.Map to create the blank map
+blank_map = sunpy.map.Map(data, header)
+
+################################################################################
+# Now we have constructed the map, we can plot it and mark important locations to it.
+
+# Initialize the plot and add the map to it
+fig = plt.figure()
+ax = plt.subplot(projection=blank_map)
+blank_map.plot(alpha=0)
```

Comment:
The need for `alpha=0` should be explained in a comment

Reply:
Added 'Plot the map with alpha=0 to provide white plot' in the comments

Reply:
If the array is all `NaN`s then I don't think you need this.